conf/messages.cy

```diff
@@ -267,10 +267,12 @@ postcode.llp.title=Beth yw cod post cyfeiriad cofrestredig eich partneriaeth ate
 #Nino page
 nino.title=Beth yw’ch rhif Yswiriant Gwladol?
 nino.hint=Mae hwn ar eich cerdyn Yswiriant Gwladol, llythyr ynghylch budd-daliadau, slip cyflog neu P60. Er enghraifft, ‘QQ 12 34 56 C’.
+nino.hint-llp=
 
 #Date of Birth page
 date-of-birth.title=Beth yw’ch dyddiad geni?
 date-of-birth.hint=Er enghraifft, 31 3 1980
+date-of-birth.hint-llp=
```

Comment:
just add the default english text here - incase if we have to deploy the service before welsh gets ready ...
seing english is not worse than seeing empty text

Reply:
OK, I added the English text, thx